src/server/routes/monthData.js

```diff
@@ -52,10 +53,10 @@ function currentMonthData(req, res, next) {
 
       next();
     })
-    .catch(error => {
+    .catch((error) => {
       console.error(`Status Code: ${error.statusCode}, Error Message: ${error.code}`);
 
-      return res.redirect('/books-music-dvds/recommendations/404');
+      return res.redirect('/books-music-movies/recommendations/404');
```

Comment:
We're not using config.baseUrl here

Reply:
True, all those redirects should point to the new 404 page, too.